go/install/install_test.go

```diff
@@ -49,3 +52,54 @@ func TestCommandLine(t *testing.T) {
 		t.Fatalf("%s", err)
 	}
 }
+
+func TestLastModifiedMatchingFile(t *testing.T) {
+	var err error
+	tc := libkb.SetupTest(t, "TestLastModifiedMatchingFile", 1)
+	defer tc.Cleanup()
+	tmpdir, err := ioutil.TempDir("", "TestLastModifiedMatchingFile")
+	defer os.RemoveAll(tmpdir)
+	require.NoError(t, err)
+	nameMatch := "blerg"
+	contentMatch := "lemon"
+	matchingContent := fmt.Sprintf("la la la\nblah blah\nblah%s a match!\n", contentMatch)
+	unmatchingContent := "la la la\nblah blah\nblah no matches\n"
```

Comment:
I noticed `ExpandEnv` got removed. Could be nice to test with os.Setenv.

Reply:
that was a mistake that got missed because of another mistake. i probably would have noticed it from the other one, but thank you for pointing this out. it's back now.